### ISSUE TITLE
Code for features page 

### ISSUE THREAD

**Issue by Swarnimashukla** [Swarnimashukla/Automatic-attendance-management-system#122]
Check for issue #107 and add elements accordingly.

In this particular issue you are supposed to code feature page(main content) check for the wireframes in resource branch

Tech Stack : MERN Stack

**Comment by aniketsingh98571** [comment]
@Swarnimashukla I am little bit confused, should we have to code the frontend part? and I cannot find the wireframe for `feature page` in resources directory. Can you briefly explain this issue...

**Comment by Swarnimashukla** [comment]
@aniketsingh98571  you can find this under resources folder in resource branch, and yes for this particular  issue front-end is needed

**Comment by keshavga1209** [comment]
I would like to try to code this page. **Please assign it to me**. I will be using React to code it.

**Comment by Swarnimashukla** [comment]
@keshavga1209  please look for issue #121  you can find the design in last comment , it will be added soon in resource branch as well

**Comment by Swarnimashukla** [comment]
@keshavga1209  what's the update regarding code ?

**Comment by Avnish5** [comment]
@Swarnimashukla Kindly assign this task to me under GSSOC 2021.

**Comment by keshavga1209** [comment]
> @keshavga1209 what's the update regarding code ?

@Swarnimashukla Sorry I was busy with college exams and projects. I will try to complete it by sunday.

**Comment by anshika208** [comment]
hey @Swarnimashukla if the issue is still pen and assignee hasnt done it may i workon the issue?

**Comment by Swarnimashukla** [comment]
@anshika208  what's the update on this task?

**Comment by anshika208** [comment]
Yes i m working on it .. was having some helath issues but from last 2 days i m wokring on it will create a pr surely by day after tomorrow.  A but of things are only left.

**Comment by Swarnimashukla** [comment]
Sure @anshika208  no issue please take your time and do take care of yourself 🙂👍

**Comment by subhangi2731** [comment]
@Swarnimashukla can I please work on this

**Comment by subhangi2731** [comment]
@Swarnimashukla please assign me this issue I want to work on this

**Comment by Swarnimashukla** [comment]
@anshika208 can you make the pr now, if done with coding?

**Comment by aakashraj01** [comment]
@Swarnimashukla is this issue still open ? Can you assign me this issue .I want to work
